### ISSUE TITLE
Don't create .d.ts if no classes available

### ISSUE THREAD

**Issue by jens-duttke** [Megaputer/dts-css-modules-loader#21]
Would it be possible to add an option, which prevents the creation of the `d.ts` file (and removes it if it exists), if no classes were found?

**Comment by c0gnize** [comment]
If the file will not be created, during importing from the absent file typescript will give classes from:
```
/**
 * Trap for `*.scss.d.ts` files which are not generated yet.
 */
declare module '*.scss' {
  var classes: any;
  export = classes;
}
```
So will not be type-checking. Are you sure you want to have this option?

**Comment by jens-duttke** [comment]
Sorry, maybe I expressed myself ambiguously.

What I mean is, if a .scss file contains no classes `dts-css-modules-loader` creates an empty file, but in that case I would prefer that no files are created, because it's not needed.

Why I need that:
I've created a Gatsby plugin, which is using your loader to generate `.d.ts` files:
https://www.npmjs.com/package/gatsby-plugin-dts-css-modules

Gatsby is based on Webpack and it's possible to extend the configuration by using plugins, which are manipulating the configuration object by injecting or removing entries.

So I'm iterating over the rules and place your loader after each occurence of the `css-loader` loader.

The downside of that is, that your loader is also used for non-module styles. It's not possible to detect if a specific `css-loader` is used for CSS modules or for "normal" CSS files, especially because it's even possible that `css-loader` switch it's behaviour at runtime based on the CSS file name by using a function for the `css-loader` options `modules.auto` or `modules.mode`.

So for me the best automatic solution would be, to only create the d.ts file, if the CSS-source file contains classes, since my "global" CSS files don't contain any classes.
